### PR TITLE
feat: permission remaining issues

### PR DESCRIPTION
**Pull request by ybzky**
close #xxx

## Pull Request Checklist

- [x] Related tickets or issues have been linked in the PR description (or missing issue).
- [x] [Naming convention](https://github.com/dream-num/univer/blob/dev/docs/NAMING_CONVENTION.md) is followed (**do please** check it especially when you created new plugins, commands and resources).
- [x] Unit tests have been added for the changes (if applicable).
- [x] Breaking changes have been documented (or no breaking changes introduced in this PR).

### CODE REVIEW

File: packages/core/src/types/interfaces/i-cell-data.ts [hunk 1]
```diff
@@ -73,6 +73,7 @@ export interface ICellMarks {
     tr?: ICellMarksStyle;
     bl?: ICellMarksStyle;
     br?: ICellMarksStyle;
+    isSkip?: boolean;
```
Comment (wzhudev) [inline packages/core/src/types/interfaces/i-cell-data.ts:76]: This property appears in 6 different places. Not smelling good. What does it mean?

Comment (ybzky) [reply packages/core/src/types/interfaces/i-cell-data.ts:76]: use to skip extension render; each extension manages its own rendering logic

Comment (ybzky) [reply packages/core/src/types/interfaces/i-cell-data.ts:76]: create a ticket optimization later and put permission on the top layer to caculate @ybzky

**Review by wzhudev — APPROVED** [review]
LGTM

**Pull request merged**